presto-docs/src/main/sphinx/release/release-0.278.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Release 0.278
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix :func:`round` returning incorrect results for edge cases.
+* Fix the compilation error when aggregation has order by clause and the input is a function.
+* Add ``selectedUser`` into Identity object.
+* Add a configuration property ``permissions.authorized-identity-selection-enable``. When it is true, the new API ``selectAuthorizedIdentity`` is called; otherwise, ``checkCanSetUser`` is called instead as before.
+* Add a new security API ``selectAuthorizedIdentity``.
+* Add an optimization which rewrites ``IF(predicate, AGG(x))`` to aggregation with mask in plan level. Controlled by system property `optimize_conditional_aggregation_enabled` and default to false.
+* Add memory limit check for HashBuilderOperator during memory revoke.
+* Add null masking for the Parquet decryption feature. When this feature is enabled and the user is denied access encrypted column, the columns will be removed in the requested schema sent to Parquet. Then it is filled out with 'NULL'  when the result is returned.
+* Add optimization for `approx_percentile` functions evaluation. Multiple `approx_percentile` functions on the same field will be combined to be one `approx_percentile` function which takes an array of percentile as arguments. The optimization is controlled by session property `optimize_multiple_approx_percentile_on_same_field` which is true by default.
+* Add optimization for outer join. Add randomized value for NULL join keys to avoid skew in NULL. Default to false.
+* Added retry with increased partition count if query fails due to low partition count.
+* Added session properties `spark_hash_partition_count_scaling_factor_on_out_of_memory` and `spark_retry_on_out_of_memory_higher_hash_partition_count_enabled` for this feature.
+* A new builtin function :func:`map_subset` is added. This function takes a map and an array of keys and returns a map with entries from the input map with keys contained in the array supplied.
```

Comment:
Just say Add function :func:\`map_subset\`, function instead of the verbose first line.